cicd/config.ts

```diff
@@ -0,0 +1,16 @@
+const accountIds: { [stageName: string]: number } = {
```

Comment:
Seems like a bad idea to hard code account IDs into this file. Should they not be included as environment settings?

Reply:
We can parameterise them with environment variables, config files, etc?
Curious why it's a bad idea? For a given organisation, these account IDs will be static. Would there be a reason to dynamically change the account for production, for example and not have that change-managed in version control?